.clang-tidy

```diff
@@ -11,9 +12,12 @@ Checks:
     cert-env33-c,
     cert-err34-c,
     cert-err*-cpp,
+    -cert-err58-cpp,
```

Comment:
I don't think we should disable this check entirely. I think it should be demoted to a warning instead. Thoughts?

Reply:
Ohhh, I just realized that I disabled this entirely. I meant to only demote it.

Will update config.

Reply:
Done.